lib/scout_apm/agent.rb

```diff
@@ -118,6 +123,8 @@ def start(options = {})
       init_logger
       logger.info "Attempting to start Scout Agent [#{ScoutApm::VERSION}] on [#{environment.hostname}]"
 
+      @recorder = create_recorder
```

Comment:
Once in agent startup, and then once in background worker start. Probably is unnecessary with further change I made to async - since it restarts the thread now if the thread stops.